src/epgsql.erl

```diff
@@ -571,3 +576,19 @@ to_map(Map) when is_map(Map) ->
     Map;
 to_map(List) when is_list(List) ->
     maps:from_list(List).
+
+%% @doc Activates TCP or SSL socket of the connection.
+%%
+%% If `{active, X}' is set in:
+%% - `tcp_opts' and the current mode is TCP or
+%% - `ssl_opts' and the current mode is SSL
```

Comment:
seems this doc needs to be updated to reflect the new location and name of this option: `socket_active`